### ISSUE TITLE
Есть подозрение что поменялся алгоритм и второй метод не работает больше

### ISSUE THREAD

**Issue by Victornovikov** [mikhailnov/getcourse-video-downloader#20]
Место где ссылка на видео
<img width="382" alt="image" src="https://github.com/user-attachments/assets/8bdee8b9-b2a0-4278-8460-cdb3d4ae1fbf">

видим что это 
<img width="241" alt="image" src="https://github.com/user-attachments/assets/01dca7eb-d702-46ca-8999-2fe205bd9b79">

data-master, но структура не такая как была раньше

<img width="238" alt="image" src="https://github.com/user-attachments/assets/d78755ff-65ad-4e3b-a72c-e9e3f6b8e1ba">

Других упоминаний data-master на странице нет

**Comment by NutsCraker** [comment]
Да, такая же проблема.... еще в одном месте появился blob, мне кажется туда видео запихали. Но как вытащить - не понятно
